src/components/furniture/Separators.vue

```diff
@@ -2,21 +2,21 @@
   <div>
     <div class="container" :style="cssStyleContainer">
       <div
-        v-if="amountOfSeparators === '1'"
+        v-if="separators.amountOfSeparators === '1'"
```

Comment:
Same here, when I see "separator**s**" , i'm expecting an array of separators, but instead I see it's an object. 

There must be a way to restructure properly in an array of objects or in an object containing an array under a kay. And put the names accordingly (singular or plural). 



I also see that